make/modules/jdk.accessibility/Lib.gmk

```diff
@@ -31,84 +31,50 @@ ifeq ($(call isTargetOs, windows), true)
   ACCESSIBILITY_SRCDIR := $(TOPDIR)/src/jdk.accessibility/windows/native
 
   ##############################################################################
-  # Setup rules to create 32/64 bit version of javaaccessbridge
-  #
-  # Parameter 1 Suffix
-  # Parameter 2 ACCESSBRIDGE_ARCH_ suffix
+  ## Build libjavaaccessbridge
```

Comment:
Is double `##` intentional?

Reply:
Well, yes and no. :-) I just copied the pattern I used elsewhere as a header to mark the output library name for `SetupJdkLibrary`. Now that you say this, I wonder why I started using `##`. Most of the places, but not all, use the double hash sign. Let's do this `##` for now as well, and then maybe I do another round of cross-makefile consistency and replace them all with single `#`.